pkg/app/app.go

```diff
@@ -335,11 +337,14 @@ func (a *App) Start() {
 	daHandler := handler.NewHandler(a.broker, a.log.Logger, a.config.Broker, clusterURL, providers, rules)
 
 	if clusterURL == "/" {
-		genericserver.Handler.NonGoRestfulMux.HandlePrefix("/", daHandler)
+		genericserver.Handler.NonGoRestfulMux.HandlePrefix("/", prometheus.InstrumentHandler("ansible-service-broker", daHandler))
```

Comment:
This may make it clear my ignorance. Could you not update `daHandler` above to something like:
```
daHandler := prometheus.InstrumentHandler(
    "ansible-service-broker",
    handler.NewHandler(a.broker, a.log.Logger, a.config.Broker, clusterURL, providers, rules),
)
```

Reply:
I bet you could, and that would probably be much more clear :) let me double check :+1: